dashboard/web/index.html

```diff
@@ -22,13 +21,25 @@
 
   <!-- Favicon -->
   <link rel="icon" type="image/png" href="favicon.png"/>
+
+  <script src="flutter.js" defer></script>
 </head>
 <body>
-<script src="main.dart.js" type="application/javascript"></script>
 <script>
-  window.flutterConfiguration = {
-    canvasKitBaseUrl: "/canvaskit/"
-  };
+  window.addEventListener('load', function() {
+    // Contact dit@ when this starts throwing warnings in development :)
+    window.flutterConfiguration = {
+      canvasKitBaseUrl: "/canvaskit/"
```

Comment:
You might want to explain what this does, i.e. uses the local build of CanvasKit rather than the CDN.

Reply:
Yes, that's a useful comment. Added.